pkg/security/x509source.go

```diff
@@ -476,18 +485,29 @@ func renewalTime(notBefore, notAfter time.Time) time.Time {
 	return notBefore.Add(notAfter.Sub(notBefore) * 7 / 10)
 }
 
+func contains(val string, list []string) bool {
+	for _, i := range list {
+		if i == val {
+			return true
+		}
+	}
+	return false
+}
+
 // isControlPlaneService returns true if the app ID corresponds to a Dapr
 // control plane service.
-func isControlPlaneService(id string) bool {
+func isControlPlaneService(id string, additionalCPServices []string) bool {
```

Comment:
Maybe this method can be moved to a shared package so the operator can use it too. This has the hardcoded ones (sentry, placement, injector) + supports the additional ones.

Reply:
shifted func to utils